src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Config/ServiceBusOptions.cs

```diff
@@ -27,6 +27,8 @@ public ServiceBusOptions()
             {
                 MaxConcurrentCalls = 16
             };
+
+            SessionHandlerOptions = new SessionHandlerOptions(ExceptionReceivedHandler);
```

Comment:
If you follow this down, the ServiceBusExtensionConfigProvider.LogExceptionReceived handler this invokes is using a log message like ""MessageReceiver error..." which isn't technically correct for sessions, right?

Reply:
The same args types passed in for the handler. I changed the message as you advised.